File: DAOLib/Models/DateCalendar.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DAOLib.Models
{
    public class DateCalendar
    {
        [Key]
        public virtual DateTime date { set; get; }
        public virtual int? day { set; get; }
        public virtual int? month { set; get; }
        public virtual int? year { set; get; }
    }
}

File: WebApp/app/Models/DateCalendar.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class DateCalendar extends Model
{
    use HasFactory;

    protected $table = 'date_calendars';

    protected $fillable = ['date', 'day', 'month', 'year'];
}
